internal/commands/logs/list_inputs.go

```diff
@@ -14,19 +14,21 @@ import (
 
 const (
 	flagType      = "type"
-	flagTypeUsage = "specify the type(s) of logs to list, available options: [auth, function, push, service, trigger, graphql, sync, schema]"
+	flagTypeUsage = `Specify the type(s) of logs to list (Default value: <none>; Allowed values: "auth", "function", "push", "service", "trigger", "graphql", "sync", "schema")`
 
 	flagErrors      = "errors"
-	flagErrorsUsage = "specify to view only error logs"
+	flagErrorsUsage = "View your Realm app's error logs"
 
-	flagStartDate      = "start"
-	flagStartDateUsage = "specify the start date to begin listing logs from"
+	flagStartDate = "start"
+	// TODO(REALMC-XXXX): Include docs link at the end of this usage: [Learn more: http://docs.link]
+	flagStartDateUsage = `Specify when to begin listing logs (Allowed format: "2006-01-02[T15:04:05.000-0700]")`
 
-	flagEndDate      = "end"
-	flagEndDateUsage = "specify the end date to finish listing logs from"
+	flagEndDate = "end"
+	// TODO(REALMC-XXXX): Include docs link at the end of this usage: [Learn more: http://docs.link]
```

Comment:
⚠️  need to reach out to docs to get a JIRA to link to

Reply:
https://docs.mongodb.com/realm/logs/cli/#view-logs-for-a-date-range